G/GlibNetworking/build_tarballs.jl

```diff
@@ -0,0 +1,73 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "GlibNetworking"
+version = v"2.74.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://download.gnome.org/sources/glib-networking/$(version.major).$(version.minor)/glib-networking-$(version).tar.xz", "1f185aaef094123f8e25d8fa55661b3fd71020163a0174adb35a37685cda613b")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+
+# We need to run some commands with a native Glib
+apk add glib-dev
+# Copied from GTK4 recipe
+ln -sf /usr/bin/glib-compile-resources ${bindir}/glib-compile-resources
+ln -sf /usr/bin/glib-compile-schemas ${bindir}/glib-compile-schemas
+# Remove gio-2.0 pkgconfig file so that it isn't picked up by post-install script.
+rm ${prefix}/lib/pkgconfig/gio-2.0.pc
+
+
+cd $WORKSPACE/srcdir
+install_license glib-networking-*/COPYING
+
+MESON_FLAGS=(--cross-file="${MESON_TARGET_TOOLCHAIN}")
+MESON_FLAGS+=(--buildtype=release)
+MESON_FLAGS+=(-Dopenssl=enabled)
+
+meson "${MESON_FLAGS[@]}" glib-networking-*/
+ninja -j${nproc} --verbose
+ninja install
+
+
+# Remove temporary links
+rm ${prefix}/bin/glib-compile-{resources,schemas}
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = filter!(p -> !Sys.iswindows(p), supported_platforms())
+platforms_windows = filter!(p -> Sys.iswindows(p), supported_platforms())
+
+
+# The products that we will ensure are always built
+products = [
+    FileProduct("lib/gio/modules/libgioenvironmentproxy.so", :libgioenvironmentproxy),
+    FileProduct("lib/gio/modules/libgioopenssl.so", :libgioopenssl)
+]
+
+products_windows = [
+    FileProduct("bin/libgioenvironmentproxy.dll", :libgioenvironmentproxy),
+    FileProduct("bin/libgioopenssl.dll", :libgioopenssl)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    HostBuildDependency("Gettext_jll")
+    Dependency("OpenSSL_jll", v"1.1.20+0"; compat="~1.1.20")
+    Dependency("Glib_jll", v"2.74.0"; compat="~2.74.0")
```

Comment:
I don't see why restricting compat to the minor version
```suggestion
    Dependency("Glib_jll"; compat="2.74.0")
```